test/src/edu/stanford/nlp/simple/TokenTest.java

```diff
@@ -3,6 +3,8 @@
 import org.junit.Test;
 import static org.junit.Assert.*;
 
+import java.util.Optional;
```

Comment:
Not seeing the reason for this being here - it compiles without this change, after all.  Was it intended to combine with the larger change at the end?